### ISSUE TITLE
urllib3 2.x complains about DEFAULT_CIPHERS

### ISSUE THREAD

**Issue by weslambert** [meeb/whoisit#20]
It looks like urllib3 2.x no longer includes DEFAULT_CIPHERS for SSL functionality, and an SSL context is preferred. 

`http_request` in `utils.py` references DEFAULT_CIPHERS and an error is thrown when attempting to using a version of requests equal to or greater than 2.30.0 and a version of urllib3 greater than 1.x.

https://github.com/meeb/whoisit/blob/main/whoisit/utils.py#L45C6-L64

Should the ssl module and ssl.create_default_context() be used to set the value?

For example, something like the following in `utils.py`. I haven't tested it.

```
import ssl
ssl_context = ssl.create_default_context()
< Optionally set ciphers >
ssl_context.set_ciphers('...')
<>
request.packages.urllib3.util.ssl_.DEFAULT_CIPHERS = ssl_context.get_ciphers()
```

Thanks!
Wes

**Comment by weslambert** [comment]
Understood. Thanks for looking into it!

**Comment by meeb** [comment]
The above commit should resolve this and be backwards as well as forward compatible. I've added a regression test as well. It'll be bundled into the next release.

**Comment by meeb** [comment]
This is bundled into the just released v2.7.0 version.

**Comment by weslambert** [comment]
Thank you!
